numba/cuda/tests/cudapy/test_inspect.py

```diff
@@ -98,6 +98,9 @@ def _test_inspect_sass(self, kernel, name, sass):
                 seen_function = True
         self.assertTrue(seen_function)
 
+        seen_lineinfo = 'line' in sass and 'inlined' in sass
```

Comment:
I worry that this might be a little fragile - would checking for `//## File` in the output rather than `line` (which might have false positives from the kernel name) and `inlined` (which might not be found if a future toolkit makes a different inlining decision) be more robust? Or even maybe checking for a match of something like:

```
r'//## File ".*/test_inspect.py", line [0-9]'
```

and `self.assertRegex()`? I'm not good at regex off the top of my head so that might need fixing up, but the intention is to check the line from the beginning up until the line number, ignoring that the paths might vary